src/current/v23.2/physical-cluster-replication-technical-overview.md

```diff
@@ -0,0 +1,57 @@
+toc: true
+docs_area: manage
+---
+
+{{site.data.alerts.callout_info}}
+{% include feature-phases/preview.md %}
+{{site.data.alerts.end}}
+
+{% include_cached new-in.html version="v23.2" %} Physical cluster replication uses two clusters with _cluster virtualization_ enabled to stream data from an active primary cluster to a passive standby cluster. Each cluster contains: a _system interface_ and an application _virtual cluster_: {% comment %}Link to Matt's work here on general cluster virtualization reference{% endcomment %}
+
+{% include {{ page.version.version }}/physical-replication/interface-virtual-cluster.md %}
+
+This separation of concerns means that the replication stream can operate without affecting work happening in the virtual cluster.
+
+### Start the replication stream
+
+Starting a physical replication stream consists of two jobs: one on the primary and one on the standby. The job on the standby cluster communicates with the primary cluster via an ordinary SQL connection and is responsible for initiating the replication stream. The stream initialization proceeds as follows:
+
+1. The standby's system interface connects to the primary cluster and starts the primary cluster's physical stream producer job.
+1. The primary cluster chooses a timestamp at which to start the physical replication stream. Data on the primary is protected from [garbage collection]({% link {{ page.version.version }}/architecture/storage-layer.md %}#garbage-collection) until it is replicated to the standby using a [protected timestamp]({% link {{ page.version.version }}/architecture/storage-layer.md %}#protected-timestamps).
+1. The primary cluster returns the timestamp and a [job ID]({% link {{ page.version.version }}/show-jobs.md %}#response) for the replication job.
+1. The standby cluster retrieves a list of all nodes in the primary cluster. It uses this list to distribute work across all nodes in the standby cluster.
+1. The initial scan runs on the primary and backfills all data from the primary virtual cluster as of the starting timestamp of the replication stream.
+1. Once the initial scan is complete, the primary then begins streaming all changes from the point of the starting timestamp.
+
+<img src="{{ 'images/v23.2/physical-rep-to.png' | relative_url }}" alt="Two virtualized clusters with system and application tenants showing the directional stream." style="border:0px solid #eee;max-width:100%" />
+
+### During the replication stream
+
+The replication happens at the byte level, which means that the job is unaware of databases, tables, row boundaries, and so on. However, when a [cutover](#cutover-and-promotion-process) to the standby cluster is initiated, the replication job ensures that the cluster is in a transactionally consistent state as of a certain point in time. Beyond the application data, the job will also replicate users, privileges, basic zone configuration, and schema changes.
```

Comment:
```suggestion
The replication happens at the byte level, which means that the job is unaware of databases, tables, row boundaries, and so on. However, when a [cutover](#cutover-and-promotion-process) to the standby cluster is initiated, the replication job ensures that the cluster is in a transactionally-consistent state as of a certain point in time. Beyond the application data, the job will also replicate users, privileges, basic zone configuration, and schema changes.
```
I wonder if it is worth searching for more instances of this across the various PRs.

Reply:
We are discussing this, happy to come back to it.